packages/lab/src/combo-box-next/ComboBoxNext.tsx

```diff
@@ -41,7 +41,7 @@ export interface ComboBoxNextProps<T>
   /**
    * Controlled prop. Controls the Input value in the Combo Box Input.
    */
-  inputValue: string;
+  inputValue?: string;
```

Comment:
This is a TS fix to the prop. Better to have a changeset

Reply:
CS added